### PR TITLE
docs([DST-570]): Added "Whats new" pages to our docs

### PR DESCRIPTION
**Pull request by sarahgm**
# Description

- possibility to generate changelogs through a script
- create directory for creating release blog posts
- adjust things around it like e.g navigation and command menu

# What should be tested?

all working, all like we decided to be? 

# Reviewers:
@marigold-ui/developer
@marigold-ui/designer
@KadaStrophe

### CODE REVIEW

**Comment by sarahgm** [comment]
Metadata can only be used with server compponents

File: docs/app/_components/Navigation.tsx [hunk 1]
```diff
@@ -47,17 +47,19 @@ export const useNavigation = (): NavigationSection[] => {
...
-              page =>
-                page.section === slug && page.subsection === subsection.slug
-            )
-            .map(page => ({
-              name: page.title,
-              href: `/${page.slug}`,
-              badge: page.badge,
-            })),
+          links: [
+            ...allContentPages
```
Comment (sebald) [inline docs/app/_components/Navigation.tsx:51]: why do we need to destructure here? :-o

**Comment by sebald** [comment]
> Metadata can only be used with server compponents

Were you able to fix it? :)

File: docs/app/_components/Navigation.tsx (hunk 1 above)
Comment (sarahgm) [reply docs/app/_components/Navigation.tsx:51]: oh we don't need it was from when I had two page types

**Comment by sarahgm** [comment]
> > Metadata can only be used with server compponents
> 
> Were you able to fix it? :)

I hope its fixed now

File: docs/scripts/build-changelog.mjs [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+    frontmatter += `title: "${packageName}"\n`;
+    frontmatter += `caption: "Have a look on the latest changes regarding ${packageName}"\n`;
+    releases.filter(release => {
+      release.badge === 'new'
+        ? (frontmatter += `badge: ${release.badge}\n`)
+        : '';
+    });
+    frontmatter += 'toc: false\n';
+    frontmatter += '---';
+    return sourceText.replace(regex, frontmatter);
```
[the remark this answers is not in the document]
Comment (sarahgm) [reply docs/scripts/build-changelog.mjs:55]: you mean just format of the file? :D

[the remark this answers is not in the document]
Comment (aromko) [reply docs/scripts/build-changelog.mjs:55]: Yeah my proposal to use blank line before return statements

File: docs/content/releases/config/eslint/whats-new.mdx [hunk 3]
```diff
@@ -0,0 +1,177 @@
+---
```
[the remark this answers is not in the document]
Comment (sarahgm) [reply docs/content/releases/config/eslint/whats-new.mdx]: yes, I renamed all generated files

**Review by KadaStrophe — APPROVED** [review]

**Review by sebald — APPROVED** [review]

**Pull request merged**